pages/common/mogrify.md

```diff
@@ -12,11 +12,11 @@
 
 `mogrify -resize {{800x600}} {{DSC*}}`
 
-- Convert all PNG images in the directory to JPEG:
+- Convert all PNGs in the directory to JPEG:
 
 `mogrify -format {{jpg}} {{*.png}}`
 
-- Halve the saturation of all image files in the current directory:
+- Half the saturation of all image files in the current directory:
```

Comment:
This doesn't make sense. `Halve` is the verb here - `Half` doesn't work the same way.

Reply:
Aaaa, good catch. Not sure what I was thinking there. 🤔 

```suggestion
- Halve the saturation of all image files in the current directory:
```